config/config.go

```diff
@@ -337,6 +351,10 @@ func DefaultRPCConfig() *RPCConfig {
 
 		Unsafe:             false,
 		MaxOpenConnections: 900,
+
+		MaxSubscriptionClients:    100,
+		MaxSubscriptionsPerClient: 5,
+		TimeoutBroadcastTxCommit:  10 * time.Second,
```

Comment:
No, because these were picked quasi randomly.

`TimeoutBroadcastTxCommit` should be less than 20s (`rpcserver.WriteTimeout`)
`MaxSubscriptionClients * MaxSubscriptionsPerClient` define total number of subscriptions. I don't think someone will want more than 500. Even 100 may be enough.